source/run/include/TGeant4.h

```diff
@@ -291,6 +292,11 @@ class TGeant4: public TVirtualMC
     virtual Double_t TrackTime() const;
     virtual Double_t Edep() const;
     virtual Double_t NIELEdep() const;
+    virtual Int_t StepNumber() const;
+    virtual Double_t TrackWeight() const;
+    virtual void TrackPolarization(Double_t &polX, Double_t &polY,
+                                              Double_t &polZ) const;
```

Comment:
Please, align either with the start of arguments list on the line above or with two spaces indention below function name start.